windows_install_build_dependencies.bat

```diff
@@ -238,37 +240,55 @@ IF /I EXIST %bazel_path% (
 )
 
 :::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::
-::::::::::::::::::::::: Python39
-set "python39_path=%opt_install_dir%\Python39\"
-set "python39_system=C:\Program Files\Python39\"
-IF /I EXIST %python39_path% (
+::::::::::::::::::::::: Python
+set "python_version=3.11.9"
+for /f "tokens=1,2 delims=." %%a in ("%python_version%") do (
+        set BEFORE_DOT=%%a
+        set AFTER_DOT=%%b
```

Comment:
Maybe we can go with names `MAJOR_VERSION` and `MINOR_VERSION` instead?
If we want to have explanation of this part maybe a comment would be a good idea?